sitemedia/scss/components/_iiif.scss

```
@@ -68,6 +68,13 @@
     // Transcription text
     .transcription {
         max-width: 60ch;
+        /* make a scrollable div; match height with image container */
+        max-height: 320px;
+        overflow: scroll;
+        @include breakpoints.for-tablet-landscape-up {
+            max-height: 680px;
+        }
+
```

CSS for max-height and overflow (disabled on mobile / with no image, based on class `.no-image`):

```suggestion
        @include breakpoints.for-tablet-landscape-up {
            /* make a scrollable div; match height with image container */
            max-height: 680px;
            overflow: scroll;
            /* no scrolling/max height when no image present */
            &.no-image {
                max-height: none;
                overflow: auto;
            }
        }
```